addons/vol-slider/addon.json

```diff
@@ -30,10 +41,21 @@
       "min": 0,
       "max": 100,
       "default": 100
+    },
+    {
+      "name": "Always show slider",
+      "id": "always",
+      "type": "boolean",
+      "default": false
```

Comment:
Are we going to change the default?

Reply:
You mean have "always show slider" turned off even for existing users of the addon? In that case, yes, that sounds good. It's an upgrade.